apps/api/src/donations/donations.controller.ts

```diff
@@ -152,7 +163,7 @@ export class DonationsController {
     return this.donationsService.create(createPaymentDto, user)
   }
 
-  @Post('create-payment-intent')
+  @Post('payment-intent')
```

Comment:
Since we're deprecating the `/create-payment-intent` endpoint in favor of `/payment-intent` have we changed the current flow of the donations to use the new endpoint? 

I'm asking to be sure that we don't interrupt the donation process until v2 donation flow is set in place.

Reply:
The `create-payment-intent` endpoint is not used for the current donation flow. 
It was merged into the api master branch in a PR a week ago https://github.com/podkrepi-bg/api/pull/410